javascript/builtins/Intl.json

```diff
@@ -1082,6 +1088,70 @@
               }
             }
           },
+          "formatToParts": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat/formatToParts",
+              "support": {
+                "webview_android": {
+                  "version_added": false
+                },
+                "chrome": {
+                  "version_added": "63",
+                  "flag": {
+                    "type": "runtime_flag",
+                    "name": "harmony-number-format-to-parts"
+                  },
+                  "notes": "Need to set the flag on the commandline via --js-flags"
+                },
+                "chrome_android": {
+                  "version_added": "63",
+                  "flag": {
+                    "type": "runtime_flag",
+                    "name": "harmony-number-format-to-parts"
+                  },
+                  "notes": "Need to set the flag on the commandline via --js-flags"
+                },
```

Comment:
Android webview seems to be missing. https://www.chromestatus.com/features/6319456309477376